common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1331,6 +1332,13 @@ def submit_problem(self, data, override_time=False):
         }
     # pylint: enable=too-many-statements
 
+    def log_closed_problem_info_for_harvard_course(self, problem_location):
+        """Temporary log to get more information"""
+        if 'HarvardX+MCB80.1x+3T2019' in problem_location:
+            log.info(
+                'Problem %s closed, close date: %s, attempts: %s/%s, is_past_due: %s',
+                problem_location, self.close_date, self.attempts, self.max_attempts, self.is_past_due()
+            )
```

Comment:
capa_base is used by all CAPA problems and this function is intended for a single course only. The function stack will grow significantly. Plus, the logs can be hit via If the condition in the same method. I don't see much utility with adding a function(unless it was being used at multiple locations)